### PR TITLE
Handle vanilla `-lang` and `-language` CLI flags for I18n

### PR DESCRIPTION
**Pull request by KawaiiYuyu**
In most cases, user would use the same language for both Terraria and TShock if the language is officially supported by this game. So we can just use those 2 flags to configure the culture.

Even if a user really want to use different langs they can also use envvars to override it.

### CODE REVIEW

**Comment by Arthri** [comment]
TShock already supports this through envvars https://github.com/Pryaxis/TShock/blob/4985c168637a70d58a511bf58156b7ad5f09960b/TShockAPI/I18n.cs#L38

File: TShockAPI/I18n.cs [hunk 1]
```diff
@@ -39,6 +41,24 @@ static CultureInfo TranslationCultureInfo
 				{
 					return new CultureInfo(overrideLang);
 				}
+
+				static CultureInfo Redirect(CultureInfo cultureInfo)
+					=> cultureInfo.Name == "zh-Hans" ? new CultureInfo("zh-CN") : cultureInfo;
+
+				if (Terraria.Program.LaunchParameters.TryGetValue("-lang", out var langArg)
```
Comment (sowelipililimute) [inline TShockAPI/I18n.cs:48]: does terraria use the same language codes as the i18n system?

**Comment by KawaiiYuyu** [comment]
> TShock already supports this through envvars
> 
> https://github.com/Pryaxis/TShock/blob/4985c168637a70d58a511bf58156b7ad5f09960b/TShockAPI/I18n.cs#L38

yep, but one need to set envvars **and** cli flags to change the whole server's language, I want to make it easier

File: TShockAPI/I18n.cs (hunk 1 above)
Comment (KawaiiYuyu) [reply TShockAPI/I18n.cs:48]: > does terraria use the same language codes as the i18n system?

we are missing some languages supported by Terraria itself, they are `it-IT`, `pt-BR` and `pl-PL`, we can add them if needed. The only exception is Chinese(Simplified), Terraria uses `zh-Hans` and Crowdin has only `zh-CN`, so I manually converted it. Other than these, I think they are compatible

Comment (hakusaro) [reply TShockAPI/I18n.cs:48]: Can we normalize this on Crowdin maybe?

**Review by hakusaro — APPROVED** [review]

**Pull request merged**